tools/WinObjC.Packaging/Package.Default.props

```diff
@@ -18,7 +18,6 @@
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
 
-    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
```

Comment:
how do i test these changes? are CI builds sufficient?

---
In reply to: [108995360](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108995360) [](ancestors = 108995360,108995058,108994710,108994382,108993288)